README.fr_FR.md

```diff
@@ -0,0 +1,82 @@
+<p align="center">
+<img src="https://media.githubusercontent.com/media/rocky-linux/branding/main/logo-text-light%402x.png" alt="Rocky Linux Logo">
+</p>
+
+<p align="center">
+<i>"En repensant aux premiers jours de CentOS ... Mon cofondateur était Rocky McGaugh. Il n'est plus avec nous, donc en tant que H / T pour lui, qui n'a jamais pu voir le succès que CentOS a connu, je vous présente. ..Rocky Linux "</i><br>
+— Gregory Kurtzer, Fondateur
+</p>
+
+## Au propos de nous
+
+Rocky Linux est un système d'exploitation d'entreprise communautaire conçu pour être 100% compatible bogue pour bogue avec Enterprise Linux, maintenant que CentOS a changé de direction.
+
+## Questions fréquemment posées
+
+> **Q:** Que voulez-vous dire, "CentOS a changé de direction?"
+
+Le projet CentOS [récemment annoncé](https://blog.centos.org/2020/12/future-is-centos-stream/) un changement de stratégie pour CentOS. Alors que CentOS existait auparavant en tant que version *en aval* de son fournisseur en amont (il reçoit des correctifs et des mises à jour après que le fournisseur en amont le fasse), il passera à une version *en amont* (test des correctifs et des mises à jour *avant* l'inclusion dans le fournisseur en amont).
+
+De plus, la prise en charge de CentOS Linux 8 a été interrompue, du 31 mai 2029 au 31 décembre 2021.
+
+> **Q:** Alors, d'où vient Rocky Linux?
+
+Rocky Linux vise à fonctionner comme une construction en aval comme CentOS l'avait fait précédemment, en construisant des versions après qu'elles aient été ajoutées au fournisseur en amont, pas avant.
+
+> **Q:** Quand va-t-il sortir?
+
+Il n'y a actuellement pas d'ETA pour la publication.
+
+> **Q:** Quelle est la vision de Rocky Linux?
+
+Une alternative **solide**, **stable** et **transparente** pour les environnements de production, développée *par* la communauté *pour* la communauté.
+
+> **Q:** Qui pilote Rocky Linux?
+
+Nous le faisons tous, Rocky Linux est un projet communautaire et le sera toujours. Rocky Linux ne sera ni vendu ni motivé par l'intérêt de l'entreprise.
+
+> **Q:** Comment puis-je m'impliquer?
+
+Veuillez consulter la section [contribution](#Contribuant) ci-dessous.
+
+## Contact
+
+| Équipe                        | Contact                       |
+|-------------------------------|-------------------------------|
+| Presse                        | outreach@rockylinux.org       |
+| Développement                 | development@rockylinux.org    |
+| Infrastructure                | infrastructure@rockylinux.org |
+| Sécurité                      | security@rockylinux.org       |
+| Web                           | web@rockylinux.org            |
+| Branding                      | brand@rockylinux.com          |
```

Comment:
Je voulais dire par là https://github.com/rocky-linux/rocky/pull/49/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L50

Reply:
done !

Reply:
@Addvilz Check now if everything is good ?